src/components/WithdrawRewards/index.tsx

```diff
@@ -105,19 +108,29 @@ function WithdrawRewards({ trigger, type, onClaim, ...rest }: WithdrawRewardsPro
                         </>
                     ) : (
                         <>
-                            <Title className="flex flex-grow justify-center pt-3">Success!</Title>
+                            <Title className="flex flex-grow justify-center pt-3">
+                              {status !== 'success' ? i18n._(t`Success!`) : i18n._(t`Congratulations!`)}
```

Comment:
whats this? why is there a difference? here I believe we only know that TX was sent to the blockchain

Reply:
in this block, if status is not success it's pending
if status is set to success, it's confirmed